.travis.yml

```diff
@@ -1,26 +1,19 @@
+language: ruby
 rvm:
-  - 1.9.3
-  - 2.0
-  - 2.1
-  - 2.2
-  - 2.3
-  - 2.4
-  - 2.5
-  - 2.6
-env:
-  - "RAILS_VERSION=3.1.0"
-  - "RAILS_VERSION=3.2.0"
-  - "RAILS_VERSION=4.0.0"
-  - "RAILS_VERSION=4.1.0"
-  - "RAILS_VERSION=4.3.0"
-  - "RAILS_VERSION=5.0.0"
-  - "RAILS_VERSION=5.1.0"
-  - "RAILS_VERSION=5.2.0"
-  - "RAILS_VERSION=6.0.0"
+  - 2.3.8
+  - 2.4.9
+  - 2.5.7
+  - 2.6.5
```

Comment:
Any reason to specify the Ruby 'teeny' version? Better to leave this without this (ie 2.3, 2.4 etc). Travis will just use the latest version it has available.

Reply:
@abrom because I can not exclude  2.3, 2.4 for rails 6.0. this configuration doesn't work
```yml
matrix:
  exclude:
    - rvm: 2.3
      gemfile: gemfiles/rails_6.0.gemfile
    - rvm: 2.4
      gemfile: gemfiles/rails_6.0.gemfile
```

Reply:
fair enough, but could you use 2.5 and 2.6  ?

Reply:
yes, we can use 2.5, 2.6

Reply:
@abrom PR updated